src/status_im2/contexts/chat/messages/content/view.cljs

```diff
@@ -91,15 +91,16 @@
                                                          context)}]))
 
 (defn user-message-content
-  [{:keys [content-type quoted-message content outgoing outgoing-status] :as message-data}
+  [{:keys [content-type quoted-message content outgoing outgoing-status edited-at] :as message-data}
    {:keys [chat-id] :as context}]
   [:f>
```

Comment:
Since you're touching this function, it looks like we don't need `:f>` here, because no hooks are being used.

Reply:
Okay, Will do it .. Thanks Icaro